index.html

```diff
@@ -177,5 +177,22 @@ <h3>Skills <img class="union-3" src="images/Union-3.png" alt=""></h3>
         </div>
         
     </section>
+    <section class="contact-form">
+        <div>
+            <h2>Contact Me</h2>
+            <p>If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding. 
+                 I’d love to help with it </p>
+        </div>
+
+        <form  class="inputs" action="#" method="post">
```

Comment:
- [ ] Please "action" value should be similar to https://formspree.io/f/{form_id}. Please follow the example here:          
[ Simple HTML forms](https://formspree.io/html/)

- [ ] Please make sure your form is submitted successfully(You can use the `network` tab in your browser dev tools to confirm)